incubator/domoticz/templates/NOTES.txt

```diff
@@ -0,0 +1,3 @@
+1. The Domoticz server can be accessed via port {{ .Values.server.httpPort }} on the following DNS name from within your cluster:
```

Comment:
Could you describe the different ways to access the server based on the ServiceType used? Take a look at https://github.com/kubernetes/charts/blob/master/stable/drupal/templates/NOTES.txt for an example.